src/status_im/ens/core.cljs

```diff
@@ -86,6 +85,23 @@
                 (multiaccounts.update/multiaccount-update
                  :preferred-name name {})))))
 
+(fx/defn remove-username
+  {:events [::remove-username]}
+  [{:keys [db] :as cofx} name]
+  (let [names  (get-in db [:multiaccount :usernames] [])
```

Comment:
indentation is a bit weird here, it should be either one space after each binding form or all expressions should be aligned